private-templates-service/adaptivecards-templating-service/src/TemplateServiceClient.ts

```diff
@@ -431,14 +427,20 @@ export class TemplateServiceClient {
     } else {
       templateInstances.push(templateInstance);
     }
-    let userResponse = await this._getUserID(authId);
-    if (!userResponse.success) return { success: false, errorMessage: userResponse.errorMessage };
+
+    // Re-create entire author array
```

Comment:
Do we need to recreate? Can use authorsList from existingTemplate and add the userResponse.result to the array if it doesn't already exist